src/igvfd/types/file_set.py

```diff
@@ -118,6 +118,73 @@ class CuratedSet(FileSet):
     schema = load_schema('igvfd:schemas/curated_set.json')
     embedded_with_frame = FileSet.embedded_with_frame
 
+    @calculated_property(
+        define=True,
+        schema={
+            'title': 'Assembly',
+            'description': 'The genome assembly to which the referencing files in the file set are utilizing (e.g., GRCh38).',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'type': 'string'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def assembly(self, request, files):
+        if files is not None:
+            assembly_values = set()
+            for current_file_path in files:
+                file_object = request.embed(current_file_path, '@@object?skip_calculated=true')
+                if file_object.get('assembly'):
+                    assembly_values.add(file_object.get('assembly'))
+        if assembly_values:
```

Comment:
I think if there are no files, this is going to result in exception, the line 141 should be within the condition `if files is not None:`

Reply:
fixed